src/gentropy/method/sumstat_quality_controls.py

```diff
@@ -86,14 +84,6 @@ def sumstat_qc_pz_check(
             SummaryStatisticsQC._calculate_logpval, t.DoubleType()
         )
 
-        window = Window.partitionBy("studyId").orderBy("studyId")
```

Comment:
This operation could be still used, you just would have to cache it after the first calculation.

Reply:
You mean over("widnow")? we don't need it anyway. I don't want to apply limit filter at all.